Sorting/insertion_sort/insertion_sort.cpp

```diff
@@ -0,0 +1,38 @@
+#include<iostream>
+
+using namespace std;
+
+void insertionSort(int arr[],int size){
+ for(int i=1;i<size;i++){
+  int j=i-1;
+  int temp=arr[i];
+  while (j >= 0 && arr[j] > temp)
+  {         
+    arr[j + 1] = arr[j];  
+    j--;  
+  }  
+  arr[j + 1] = temp; 
+ }
+}
+void display(int arr[],int size){
+ for(int i=0;i<size;i++){
+  cout<<arr[i]<<"\t";

```

Comment:
Hey @hnishal I meant space here instead of `"\t"` this -> `cout << arr[i] << " "`. The rest is well written!

Reply:
ahhh alrighttt